cli/exp_scaletest.go

```diff
@@ -525,6 +525,8 @@ func (r *RootCmd) scaletestCreateWorkspaces() *clibase.Cmd {
 
 		useHostUser bool
 
+		parameterFlags workspaceParameterFlags
```

Comment:
FYI This may also pull `--build-options`, not sure if you want to support them too. Probably not.

Reply:
I don't see a need at present. We don't rebuild workspaces in this command, only create.